### ISSUE TITLE
AttributeError: 'SubprocessStreamChannel' object has no attribute '_last_ping'

### ISSUE THREAD

**Issue by martinpitt** [cockpit-project/cockpit#19323]
I was testing cockpit-navigator and file uploading. On Fedora 38,

 - `sudo dnf install cockpit-navigator`
 - Upload a large file (I tried a 216 MB video file) to the cockpit VM with navigator

This results in a bridge crash, and thousands of these journal messages:
```
asyncio-ERROR: Exception in callback _Transport._write_ready()
handle: <Handle _Transport._write_ready()>
Traceback (most recent call last):
  File "/usr/lib64/python3.11/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/lib/python3.11/site-packages/cockpit/transports.py", line 189, in _write_ready
    self._remove_write_queue()
  File "/usr/lib/python3.11/site-packages/cockpit/transports.py", line 197, in _remove_write_queue
    self._protocol.resume_writing()
  File "/usr/lib/python3.11/site-packages/cockpit/channel.py", line 405, in resume_writing
    if self._last_ping is not None:
       ^^^^^^^^^^^^^^^
AttributeError: 'SubprocessStreamChannel' object has no attribute '_last_ping'
```

Also tracked in RHEL: https://issues.redhat.com/browse/RHEL-5341

**Comment by martinpitt** [comment]
I reproduced that in a test case in my branch: https://github.com/martinpitt/cockpit/tree/file-upload

**Comment by martinpitt** [comment]
I sent an initial fix to #19325, but it needs more work.
